tests/test_hydraulic_power.py

```diff
@@ -298,12 +299,8 @@ def energy_system_options(self):
 
         # due to non linearity, every timestep on new linearized line, a doubled mass flow should
         # result in more than doubled hydraulic power
-        np.testing.assert_array_less(
-            (pipe_hp[0] + 1e-6) * (pipe_mass[1] / pipe_mass[0]), pipe_hp[1]
-        )
-        np.testing.assert_array_less(
-            (pipe_hp[1] + 1e-6) * (pipe_mass[2] / pipe_mass[1]), pipe_hp[2]
-        )
+        np.testing.assert_array_less(pipe_hp[0] * (pipe_mass[1] / pipe_mass[0]), pipe_hp[1] + 1e-6)
```

Comment:
would prefer if you add a variable tol=1e-6, such that it is clear that all those places where you add 1e-6 it is tolerance.

Reply:
Agree. Code updated